drivers/audio/cxd56.c

```diff
@@ -2768,10 +2799,21 @@ static int cxd56_configure(FAR struct audio_lowerhalf_s *lower,
           {
             /* Set the mic gain */
 
+            /* mic_gain range : 0 ~ 210 (= 150 + 60) */
+
             priv->mic_gain = caps->ac_controls.hw[0];
             audinfo("    Mic gain: %d\n", priv->mic_gain);
 
-            /* TODO: How to set individual mic gains?  */
+            if (priv->state != CXD56_DEV_STATE_OFF &&
+                priv->state != CXD56_DEV_STATE_STOPPED )
```

Comment:
Please fix to pass in the CI

Reply:
@SPRESENSE  
nuttx/drivers/audio/cxd56.c:2808:55: error: Space precedes right parenthesis